### DISCUSSION TITLE
Would there be a release for dotnet/sdk:8.0-bullseye?

### DISCUSSION THREAD

**Discussion by samruddhikhandale** [dotnet/dotnet-docker#5002] [Q&A] [answered]
Hi 👋 

Thanks for quick release of dotnet images, appreciate it ✨ 

From https://hub.docker.com/_/microsoft-dotnet-sdk/, I don't see release for tags `8.0-bullseye` and `8.0-bullseye-slim`. Would they be released anytime soon?

**Comment by ghost** [comment]
I couldn't figure out the best area label to add to this issue. If you have write-permissions please help me learn by adding exactly one area label.

**Comment by lbussell** [comment, marked as answer]
Hi @samruddhikhandale, according to our [supported platforms documentation](https://github.com/dotnet/dotnet-docker/blob/main/documentation/supported-platforms.md#linux), we support the latest stable version of Debian at the time of a .NET major release as well as any new stable Debian versions as they are released. Since the latest stable version of Debian is Bookworm, we won't be offering Bullseye images for .NET 8.

**Reply by samruddhikhandale** [reply]
Thanks for the clarification!
